op-acceptance-tests/tests/base/chain/init_test.go

```diff
@@ -8,6 +8,6 @@ import (
 
 func TestMain(m *testing.M) {
 	presets.DoMain(m,
-		presets.WithMinimal(),
+		presets.WithSingleChainMultiNode(),
```

Comment:
Should we be using this for _all_ base tests? Does it give us extra coverage in existing tests?

Reply:
Quite possibly, yes!

Also, this is how you'd mark this test to be skipped on external devnets:


```suggestion
		presets.WithSingleChainMultiNode(),
		presets.WithCompatibleTypes(compat.SysGo),
```